README.md

```diff
@@ -72,6 +72,29 @@ Aggregate is built using Gradle and Gretty, but we strongly recommend you use [I
 
 1. You should now be able to browse [http://localhost:8080](http://localhost:8080)
 
+### Deploy to AppEngine
+
+1. Follow part of the official instructions for [Installing on AppEngine (Cloud)](http://docs.opendatakit.org/aggregate-install/#installing-on-app-engine). Stop after Google configures the server, and before the tutorial.
+
+1. Press the + button to add a `Gradle` configuration
+
+    * Name: `gaeUpdate` (or whatever you'd like)
+    * Gradle project: `odk-aggregate`
+    * Tasks: `gaeUpdate`
+    * Arguments: `-PgaeAppName=%yourAppName% -PgaeAppId=%yourAppId% -PgaeEmail=%yourEmail%`
+    * (change `%yourAppName%`, `%yourAppId%` and `%yourEmail%` with what you used in step 1)
```

Comment:
Instead of `gaeAppName`, this should probably be `aggregateInstanceName` to match the installer. 

It's what is prepended to the ODK Aggregate as the realm string in security.properties and changing it will invalidate all passwords.

<img width="691" alt="screen shot 2017-11-20 at 11 04 49" src="https://user-images.githubusercontent.com/32369/33036247-a60ea886-cde2-11e7-9b65-e07e2c018c24.png">